latest/index.bs

```diff
@@ -153,7 +153,8 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
     │   ...                   # which is a folder containing chunk files which compose the array.
     ├── n                     # The name of the array is arbitrary with the ordering defined by
     │   │                     # by the "multiscales" metadata, but is often a sequence starting at 0.
-    │   │
+    │   │                     # All arrays must have the same datatype and the same number of dimensions.
+    |   | 
```

Comment:
Minor point: Looks like 2 lines have changed because whitespace was added to the "blank" line. `    |   | ` Just adding 1 line is preferable.